ui/components/app/advanced-gas-controls/index.scss

```diff
@@ -0,0 +1,44 @@
+.advanced-gas-controls {
+  &__row {
+    margin-bottom: 20px;
+
+    &-heading {
```

Comment:
personal preference: I prefer everything that we use the & with to be some kind of block, instead of a partial name block for the ease of making the mental leap that these styles belong to the &__row-heading block.